zarr/tests/test_storage.py

```diff
@@ -930,6 +940,31 @@ def test_pickle(self):
             pickle.dumps(store)
 
 
+@unittest.skipIf(pymongo is None, 'test requires pymongo')
+class TestMongoDBStore(StoreTests, unittest.TestCase):
+
+    def create_store(self):
+        # TODO: this is the default host for MongoDB on Travis,
+        # we probably want to generalize this though
```

Comment:
Is this TODO something you want to address before merge? Would we ever need to do something different?

Reply:
Right. I think the more important question is how do you want to handle these tests when a mongodb or redis server is not available. How should we decide when to skip these tests?